apps/src/tutorialExplorer/tutorialSet.jsx

```diff
@@ -13,18 +24,61 @@ const TutorialSet = React.createClass({
     disabledTutorials: React.PropTypes.arrayOf(React.PropTypes.string).isRequired
   },
 
+  getInitialState() {
+    return {
+      showingDetail: false,
+      chosenItem: null
+    };
+  },
+
+  tutorialClicked(item) {
+    this.setState({showingDetail: true, chosenItem: item});
+  },
+
+  tutorialDetailClosed() {
+    this.setState({showingDetail: false, chosenItem: null});
+  },
+
+  changeTutorial(direction) {
+    const index = this.props.tutorials.indexOf(this.state.chosenItem);
+    let nextItem = null;
+    if (direction === "next" && index < this.props.tutorials.length - 1) {
+      nextItem = this.props.tutorials[index + 1];
+    } else if (direction === "previous" && index > 0) {
+      nextItem = this.props.tutorials[index - 1];
+    }
```

Comment:
Could potentially just have your param be a delta instead of a string direction, so instead of `changeTutorial("previous")` you would have `changeTutorial(-1)`.

Then I think you can do:
```
const nextItem = this.props.tutorial[index + delta];
if (nextItem) {
  this.setState
}
```